schema/image-index-schema.json

```diff
@@ -1,12 +1,12 @@
 {
   "description": "OpenContainer Image Manifest List Specification",
   "$schema": "http://json-schema.org/draft-04/schema#",
-  "id": "https://opencontainers.org/schema/image/manifest-list",
+  "id": "https://opencontainers.org/schema/image/image-index",
```

Comment:
Here's [another place][1] where “image index” seems over specific.  Can we follow your  `…vnd.oci.image.index…` media type (which does not double up on `index` and use `…/image/index` here (and the other `id`s you touch)?

[1]: https://github.com/opencontainers/image-spec/pull/546#discussion_r99223558